lightkurve/search.py

```diff
@@ -46,13 +47,41 @@ def __init__(self, table=None):
             self.table = Table()
         else:
             self.table = table
-
-    def __repr__(self):
+            if len(table) > 0:
+                self._add_columns()
+
+    def _add_columns(self):
+        """Adds user-friendly ``idx`` and ``collection`` columns.
```

Comment:
MAST actually returns "collections" of files, so this is a meaningful term in this framework. We should call it something else.